translations/en_GB/atrium-translations-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionOptionalAssertion.kt

```diff
@@ -6,4 +6,6 @@ import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
 
 enum class DescriptionOptionalAssertion(override val value: String) : StringBasedTranslatable {
     EMPTY("empty"),
+    VALUE("value"),
+    PRESENT("present")
```

Comment:
see reporting in the issue, this should be used if the Optional is not present. Hence rename it to something like the following with the corresponding text: `IS_NOT_PRESENT("❗❗ is not present")`